source/D2Game/src/GAME/Game.cpp

```diff
@@ -1697,7 +1697,7 @@ void __fastcall GAME_EndGame(int32_t nClientId, int32_t a2)
                 while (CLIENT_GetSaveHeader_6FC34350(pClient));
             }
 
-            D2GAME_PACKETS_SendPacket0x4F_StartMercList_6FC3C6F0(pClient, 6u);
+            D2GAME_PACKETS_SendPacket0x4F_UpdateClientState(pClient, 6u);
```

Comment:
Seems like 6 is player left?

Reply:
Yes, but as mentionned in the other comment, I'm not even 100% certain that it actually is the same enum used here, but it is indeed similar.